src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -26,7 +25,7 @@ public Movie add(Movie movie) {
             if (transaction != null) {
                 transaction.rollback();
             }
-            throw new DataProcessingException("Can't insert movie " + movie, e);
+            throw new DataProcessingException("Can't insert cinema hall " + movie);
```

Comment:
Include the original exception as a cause in the `DataProcessingException` to provide more context about the error. Use `throw new DataProcessingException("Can't insert movie " + movie, e);`.